deps/v8/include/v8.h

```diff
@@ -1940,7 +1940,7 @@ class V8_EXPORT ScriptCompiler {
    */
   class V8_EXPORT StreamedSource {
    public:
-    enum Encoding { ONE_BYTE, TWO_BYTE, UTF8 };
+    enum Encoding { ONE_BYTE, TWO_BYTE, UTF8, WINDOWS_1252 };
```

Comment:
This is just fine :+1: Generally, adding new enum values and leaving existing ones unchanged is ABI-compatible, the only risk is that if new enum values are returned from a function, the caller might just not expect that value to be possible (but that’s not the case here, and isn’t an ABI concern, only API)